client/landing/stepper/declarative-flow/internals/types.ts

```diff
@@ -139,6 +139,12 @@ export type Flow = {
 	 * A hook that is called in the flow's root at every render. You can use this hook to setup side-effects, call other hooks, etc..
 	 */
 	useSideEffect?: UseSideEffectHook< ReturnType< Flow[ 'useSteps' ] > >;
+
+	/**
+	 * Temporary hook to allow gradual migration of flows to the globalised/default event tracking.
+	 * IMPORTANT: This hook will be removed in the future.
+	 */
+	use__Temporary__ShouldTrackEvent?: ( event: keyof NavigationControls ) => boolean;
```

Comment:
WDYT about making this a boolean since it's just meant as a stepping stone? `__useBuiltInTracks` or such?

Reply:
It will change the direction that I have in mind. My intention is to piece-meal the migration, event by event. The alternative, a single boolean, would mean handling all events at once, per flow. I'm going per-event first. See a usage: https://github.com/Automattic/wp-calypso/pull/93136

In the end, we can either remove (so default logging always) or turn into a boolean once everything migrates.

Reply:
OK sounds great.